zuul.d/projects.yaml

```diff
@@ -118,15 +118,12 @@
         - cluster-api-provider-openstack-image-build:
             branches: master
 
-#NOTES: Should be updated when switch to official project
 - project:
-    name: zhongjun2/bosh-huaweicloud-cpi-release
+    name: cloudfoundry-incubator/bosh-huaweicloud-cpi-release
     check:
-      jobs:
-        - bosh-huaweicloud-cpi-release-acceptance-test
-    check-generic-cloud:
       jobs:
         - bosh-acceptance-test-huaweicloud
+        - bosh-huaweicloud-cpi-release-validator-test
```

Comment:
According [jobs-naming-notations](https://github.com/theopenlab/openlab-zuul-jobs#jobs-naming-notations), jobs name should be 
```
- bosh-huaweicloud-cpi-release-acceptance-test-huaweicloud
- bosh-huaweicloud-cpi-release-validator-test-huaweicloud
```